tests/test_pysat_cardinality.py

```diff
@@ -125,6 +127,26 @@ def test_pysat_card_implied(self):
             cp.Model(c).solve("pysat")
             self.assertTrue(c.value())
 
+    def test_pysat_support_negative_coefficients(self):
+        bvs = cp.boolvar(3)
+        # this is linearized to `a+b-c>0`, which previously wasn't rewritten to the cardinality constraint `a+b+~c>1`
+        c = sum(bvs[1:]) > bvs[0]
+        s = cp.SolverLookup.get("pysat")
+        s += c
+        self.assertTrue(s.solve())
+
+    def test_only_positive_coefficients(self):
+        a, b, c = [cp.boolvar(name=n) for n in "abc"]
+        only_pos = only_positive_coefficients([Operator("wsum",[[1,1,-1],[a,b,c]]) > 0])
+        self.assertEqual(str([Operator("sum",[a, b, ~c]) > 1]), str(only_pos))
+
+    def test_pysat_aggregate_sum_sub_expressions(self):
+        bvs = cp.boolvar(3)
+        c = bvs[0] > sum(bvs[1:])
+        s = cp.SolverLookup.get("pysat")
+        print(c, "==>", s.transform(c))
+        s += c
+        s.solve()
 
```

Comment:
Can you also add some tests where the sum contains a mix of integer and boolean variables?